apps/dash-oil-gas-ternary/README.md

```diff
@@ -0,0 +1,51 @@
+## dash-oil-gas-ternary
+
+`dash-oil-gas-ternary` creates a dashboard for mineral composition evaluations from natural gas wells.  
+By pairing a geographic map of well locations with ternary diagram, this app helps geologists to find out results of gas wells with relationship to formations or rock types.
+This is a demo of Dash interactive Python framework developed by [Plotly](https//plot.ly/).
+
+![Animated](assets/Screencast.gif)
+
+## Screenshots
+![initial](assets/Screenshot.png)
+
+## Requirements
+We suggest you to create a separate virtual environment running Python 3 for this app, and install all of the required dependencies there. Run in Terminal/Command Prompt:
+
+```
+git clone https://github.com/plotly/dash-oil-gas-ternary.git
```

Comment:
@shammamah told me these repo will be archived so probably put the monorepo link here and have them cd into dash-sample-apps/apps/dash-oil-gas-ternary

Reply:
I'd add a migration/deprecated notice in original repo readme.